app/static/js/app.js

```diff
@@ -293,6 +293,24 @@ function setCursor(cursor, save = true) {
   }
 }
 
+function setFullScreen() {
+  const remoteScreen = document.getElementById("remote-screen");
+  const remoteScreenImg = document.getElementById("remote-screen-img");
+  if (!remoteScreen.dataset.onfullscreenchange) {
```

Comment:
It took me a while to understand the body of this `if` clause. I think it would be easier if we just assigned the event handler at the global level with all the others at the bottom of app.js. Suggested rewrite:

https://github.com/djclueless/tinypilot/pull/3

Reply:
Thats because the event is tied to "app" and it isn't visible or focused in full screen mode. I changed it to document, but that now interferes with pasting.